packages/react-microlink/src/index.js

```diff
@@ -5,32 +5,39 @@ import {CardWrap, CardMedia, CardContent, CardEmptyState} from './components/Car
 import {getUrlPath, someProp, createApiUrl} from './utils'
 
 class Microlink extends Component {
+  constructor (props) {
+    super(props)
+    this.setData = this.setData.bind(this)
```

Comment:
Why not an arrow function to remove necessity of binding?

Reply:
oh, nice. It's possible because we have `transform-class-properties` plugin 👌